dvc/commands/experiments/rename.py

```diff
@@ -0,0 +1,83 @@
+import argparse
+import logging
+
+from dvc.cli.command import CmdBase
+from dvc.cli.utils import append_doc_link
+from dvc.exceptions import InvalidArgumentError
+from dvc.ui import ui
+
+logger = logging.getLogger(__name__)
+
+
+class CmdExperimentsRename(CmdBase):
+    def check_arguments(self):
+        if not any(
+            [
+                self.args.rev,
+            ]
+        ) ^ bool(self.args.experiment):
+            raise InvalidArgumentError(
+                "Either provide an `experiment` argument, or use the "
+                "`--rev` or `--all-commits` or `--queue` flag."
+            )
+
+    def run(self):
+        from dvc.utils import humanize
+
+        self.check_arguments()
+
+        renamed = self.repo.experiments.rename(
+            exp_name=self.args.experiment,
+            new_name=self.args.name,
+            rev=self.args.rev,
+            git_remote=self.args.git_remote,
+        )
+        if renamed:
+            ui.write(f"Renamed experiments: {humanize.join(map(repr, renamed))}")
+        else:
+            ui.write("No experiments to rename.")
+
+        return 0
+
+
+def add_parser(experiments_subparsers, parent_parser):
+    EXPERIMENTS_REMOVE_HELP = "Rename experiments."
+    experiments_rename_parser = experiments_subparsers.add_parser(
+        "rename",
+        parents=[parent_parser],
+        description=append_doc_link(EXPERIMENTS_REMOVE_HELP, "exp/remove"),
+        help=EXPERIMENTS_REMOVE_HELP,
```

Comment:
Change this to `EXPERIMENTS_RENAME_HELP`. Also, create an issue in dvc.org at some point